osbenchmark/utils/dataset.py

```diff
@@ -106,7 +113,7 @@ def read(self, chunk_size: int):
         return vectors
 
     def seek(self, offset: int):
-
+        # load file first before seek
         if offset < self.BEGINNING:
```

Comment:
Should there be a `self._load()` here ?

Reply:
Updated. We don't have to load for seek. I removed that comment.